src/useGet.test.tsx

```diff
@@ -645,6 +645,63 @@ describe("useGet hook", () => {
     });
   });
 
+  describe("originalResponse", () => {
+    it("should return original response when flag is enabled", async () => {
+      nock("https://my-awesome-api.fake")
+        .get("/")
+        .reply(200, { oh: "my god 😍" });
+
+      const MyAwesomeComponent = () => {
+        const { loading, response } = useGet({
+          path: "/",
+          originalResponse: true,
```

Comment:
Why this flag? I would expect to always send the response

Reply:
I assumed that returning response was left out on purpose since it's part of the TS interface, but not actually returned anywhere. If that's not the case, I can definitely remove the flag.

I am a bit apprehensive about always returning the entire response by default because it can have performance and memory-usage implication. Returning the response means that the data is duplicated (once in parsed form and once as a Blob in response).

Another option I considered was to just return the response headers separately (and continue returning null for response). That will solve a common request for headers but it'll not be a very clean API.

What do you think?

Reply:
I'm not too afraid about memory over there, will be garbage collect quite quickly if you don't consume it and we can still add the flag later if we really have some memory issues.

I actually like the `responseHeaders` option, especially if we can bring back some types over there 😀 But I'm not against adding the `response` for advanced usages, even if this can't really be typesafe. This is always good to have the information around.

Reply:
Cool. I'll remove the flag then. It'll be much cleaner then.